pages/common/cmake.md

```diff
@@ -1,13 +1,28 @@
 # cmake
 
-> Cross-platform build system generator.
-> It generates Makefiles, Visual Studio projects or others, depending on the target system.
+> Cross-platform build automation system, that generates recipes for native build systems.
 > More information: <https://cmake.org/cmake/help/latest/manual/cmake.1.html>.
 
-- Generate a Makefile and use it to compile a project in the same directory as the source:
+- Generate a build recipe in the current directory with `CMakeLists.txt` from a project directory:
 
-`cmake && make`
+`cmake {{path/to/project_directory}}`
 
-- Generate a Makefile and use it to compile a project in a separate "build" directory (out-of-source build):
+- Generate a build recipe, build type setted to `Release` with CMake variable:
 
-`cmake -H. -B {{build}} && make -C {{build}}`
+`cmake {{path/to/project_directory}} -D CMAKE_BUILD_TYPE=Release`
+
+- Build artifacts with the generated recipe in `build_directory`:
+
+`cmake --build {{path/to/build_directory}}`
+
+- Install the build artifacts into `/usr/local/` and strip debugging symbols:
+
+`cmake --install {{path/to/build_directory}} --strip`
+
+- Install the build artifacts using the custom prefix for paths:
```

Comment:
```suggestion
- Install the build artifacts into a custom directory:
```

Reply:
I'd like to avoid this substitution.
- "Directory" borns suggestion that artifacts will be placed in the directory. But actually usually project hierarhy will be created in the prefix dir.
- Prefix is not just «let install it here», it may cause project not actually be installed in current system.

Reply:
@bergentroll Okay!  That's great that you've thought about it so much :D
I don't have a strong opinion.